src/@types/assets/window.d.ts

```diff
@@ -27,5 +27,25 @@ declare global {
 		wpParselyHooks?: _Hooks;
 		wpParselyPostsStatsResponse: string;
 		wpParselyEmptyCredentialsMessage: string;
+
+		/**
+		 * Jetpack Editor Initial State.
+		 * This is required for the Excerpt Generator feature to know if Jetpack
+		 * AI Content Lens is available and enabled.
+		 *
+		 * @since 3.13.0
+		 *
+		 * @see https://github.com/Automattic/jetpack/blob/4eb6a42833879b30aa2a7f4c82e44fc094307de3/projects/plugins/jetpack/extensions/plugins/ai-content-lens/editor.js#L16
+		 */
+		Jetpack_Editor_Initial_State?: {
```

Comment:
I just realised that we already have a `window.d.ts` file for declaring `window` types, so I ended up moving the `Jetpack_Editor_Initial_State` here, but also introduced a comment further explaining it.

Reply:
Yes, that's great!